rasa_nlu/server.py

```diff
@@ -429,13 +450,12 @@ def unload_model(self, request):
         if 'all' in pre_load:
             pre_load = router.project_store.keys()
         router._pre_load(pre_load)
-
     rasa = RasaNLU(
         router,
         cmdline_args.loglevel,
         cmdline_args.write,
         cmdline_args.num_threads,
-        cmdline_args.token,
+        get_token(),
```

Comment:
I would change that to `get_token(cmdline_args.token)`. Should be saver and also more reusable than accessing the one from `__main__`